PodcastGenerator/.gitignore

```diff
@@ -1,7 +1,2 @@
-categories.xml
-config.php
-freebox-content.txt
 rsscache_*
-feed.xml
 themes/testtheme
-buttons.xml
```

Comment:
Why are there changes to the .gitignore?

Reply:
those file was moved to appdata and i created links to them in appdata then i ignore files in appdata